### ISSUE TITLE
Error: Node in node in node

### ISSUE THREAD

**Issue by alexspirtmsu** [iVis-at-Bilkent/cytoscape.js-expand-collapse#39]
1. Initialize cytoscape:
```
cy = cytoscape({
   container: document.getElementById('cy'),
   elements:[
      {'data': {'id': '1'}, 'group': 'nodes'},
      {'data': {'id': '2', 'parent': '1'}, 'group': 'nodes'},
      {'data': {'id': '3', 'parent': '2'}, 'group': 'nodes'},
   ]

});
```
2. Initialize expand-collapse extension with default options.
3. Collapse node '2'.
4. Move collapsed node.
5. Try to expand collapsed node and it leads to error:

> Uncaught TypeError: Cannot read property 'x' of undefined. expandCollapseUtilities.js:13

**Comment by ugurdogrusoz** [comment]
Solving this particular issue (https://github.com/iVis-at-Bilkent/newt/issues/53) might resolve this one.

**Comment by kinimesi** [comment]
@alexspirtmsu should be fixed by 05552cd9de6d6e6c4401fe76bc2b3201e5d0fd56 in unstable branch. Can you verify and close the issue? Thanks.

**Comment by alexspirtmsu** [comment]
It works. Thanks!!!
